src/agent/src/agent.cpp

```diff
@@ -113,7 +115,7 @@ void Agent::Run()
                               "FetchCommands");
 
     m_taskManager.EnqueueTask(m_communicator.StatefulMessageProcessingTask(
-                                  [this](const int numMessages)
+                                  [this](const size_t numMessages)
```

Comment:
The variable named should now be messageSize

Reply:
done